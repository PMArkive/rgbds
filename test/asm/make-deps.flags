-M - -MT preserve$dollars$$ -MQ escape$dollars$$
