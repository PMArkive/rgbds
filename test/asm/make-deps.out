preserve$dollars$$ escape$$dollars$$$$: make-deps.asm
